client/src/components/documents/Document.tsx

```diff
@@ -503,11 +504,116 @@ export default function Document({ user }: DocumentProps) {
                         </DialogPanel>
                     </Dialog>
 
+                </div>
+
+                <div className="flex flex-col space-y-2 ">
+                    <div className="flex flex-row">
+                        <h3 className="text-l font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Connections</h3>
+                        {canEdit && <i className="ml-2" /*onClick={() => setIsOpen(true)}*/><RiEditBoxLine className="text-2xl text-tremor-content-strong dark:text-dark-tremor-content-strong " /></i>}
+                    </div>
+
+                    <div className="flex flex-col lg:flex-row ">
+
+                        <div className="flex w-full h-full items-center justify-between mb-2">
+                            <Accordion className="w-full mr-6 mb-6">
+                                <AccordionHeader className="text-sm font-medium text-tremor-content-strong dark:text-dark-tremor-content-strong">Direct connections</AccordionHeader>
```

Comment:
it's better to separate the header with the content. including background color and a border. I checked on both theme (I mean dark and light)

Reply:
Sorry, what do you mean with "header"?